docs/cmd/kn_service_update.md

```diff
@@ -70,6 +70,7 @@ kn service update NAME
       --requests-memory string        DEPRECATED: please use --request instead. The requested memory (e.g., 64Mi).
       --revision-name string          The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
       --scale int                     Minimum and maximum number of replicas.
+      --scale-init int                Initial scale for a service
```

Comment:
Perhaps mention here that the range is 0..some positive number, I imagine limited by vendor or is it limited in API or just the max positive int. 

Also, since max value of int would be very large, perhaps a recommended sensible value is better... Not clear but need to say here that 0..positive number is range since folks might assume 1 is minimum.

Reply:
Actually I thought that this option was to be used with service create only so I didn't keep in mind that update will use it too. Therefore I haven't added test cases for service update either, if you will see. I just had this discussion with Navid yesterday and I'm adding service update test cases as well.

Also, there will be some changes needed for update to work too. I will make them by EOD.